package.json

```diff
@@ -14,7 +14,7 @@
     "format": "prettier --write \"**/*.{ts,tsx,md}\"",
     "storybook": "pnpm run --parallel --recursive storybook:dev",
     "storybook:build": "storybook build",
-    "migrations": "cd apps/builder && pnpm migrations --dev",
+    "migrations": "cd apps/builder && pnpm migrations migrate --dev",
```

Comment:
When i run migrations in local, it needs either `--dev` or `--force` flag as mandatory. I am manually adding when i run migrations till now.

Reply:
@istarkov

Reply:
Prev command already had --dev. nothing has changed except you add `migration`

Reply:
Apologies, I mixed up. It was the missing _migrate_ command itself

Reply:
we use it not only with migrate ;)

Reply:
the only reason it exists is —dev flag ;)